pkgs/development/python-modules/yanc/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, pythonOlder, fetchPypi, nose }:
+
+buildPythonPackage rec {
+  pname = "yanc";
+  version = "0.3.3";
+
+  # Tests fail on Python>=3.5. See: https://github.com/0compute/yanc/issues/10
+  disabled = !(pythonOlder "3.5");
```

Comment:
does "tests fail" mean that the package is broken? Otherwise you could simply write `doCheck = pythonOlder "2.5"`.

Reply:
I don't know. :slightly_smiling_face: I'm not familiar with the package but I would assume the package doesn't work as expected if the tests fail, thus I'd disable the package. I've made a bug report upstream (the link in the comment): https://github.com/0compute/yanc/issues/10